.code-samples.meilisearch.yaml

```diff
@@ -3,19 +3,52 @@
 # the documentation on build
 # You can read more on https://github.com/meilisearch/documentation/tree/master/.vuepress/code-samples
 ---
-authorization_header_1: |-
-  client = try MeiliSearch(host: "http://localhost:7700", apiKey: "masterKey")
-  client.getKeys { result in
-      switch result {
-      case .success(let keys):
-          print(keys)
-      case .failure(let error):
-          print(error)
-      }
+async_guide_filter_by_date_1: |-
+  client.getTasks(params: TasksQuery(afterEnqueuedAt: date)) { result in
```

Comment:
Should the date not be something like `2020-10-11T11:49:53.000Z` instead of `date`?